spring-integration-rmi/src/main/java/org/springframework/integration/rmi/RmiProxyFactoryBeanConfigurer.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.rmi;
+
+import org.springframework.remoting.rmi.RmiProxyFactoryBean;
+
+/**
+ * @author Gary Russell
```

Comment:
How about some general JavaDocs
```
A callback to provide additional configuration for the {@link RmiProxyFactoryBean} before proxy is created.
```
?

OTOH why this callback isn't inner class in the `RmiOutboundGateway`?